CommonTools/PileupAlgos/src/PuppiAlgo.cc

```diff
@@ -151,6 +154,8 @@ void PuppiAlgo::computeMedRMS(const unsigned int &iAlgo,const double &iPVFrac) {
     double lAdjust = double(lNPV)/double(fPupsPV.size()+fNCount[iAlgo]);
     if(lAdjust > 0) fMedian[iAlgo] -= sqrt(ROOT::Math::chisquared_quantile(lAdjust,1.)*fRMS[iAlgo]);
 
+    // std::cout << "*change2* -- iAlgo = " << iAlgo << ", fRMSEtaSF = " << fRMSEtaSF << ", fMedEtaSF = " << fMedEtaSF << ", fRMS[iAlgo] = " << fRMS[iAlgo] << ", fMedian[iAlgo] = " << fMedian[iAlgo] << std::endl;
```

Comment:
only commented out couts and empty lines have changed. Were these changes really necessary?